wm/position.go

```diff
@@ -0,0 +1,20 @@
+package wm
+
+import (
+	"fyne.io/fynedesk"
+	"fyne.io/fynedesk/theme"
+)
+
+// PositionForNewWindow returns the suggested position for a new window of the given geometry.
+// The screen list hints at available space, but normally list.Active() is the best.
+func PositionForNewWindow(x, y int, w, h uint, screens fynedesk.ScreenList) (int, int, uint, uint) {
+	if x != 0 && y != 0 {
```

Comment:
Instead of this check - lets just test for a specified position like so:
hints, err := icccm.WmNormalHintsGet(c.wm.X(), c.win)
if err == nil {
    if hints.Flags & icccm.SizeHintPPosition > 0 && hints.Flags & icccm.SizeHintUSPosition > 0 {
        *iterate screens and check to make sure x,y instersect somewhere and isn't broken*
        if !broken { return } else we will continue on and let the top left positioning happen.
    }
}

Reply:
I have found a way of doing this I think

Reply:
This is fixed but in the X code as those properties are not exposed. This is now called conditionally - if it was not positioned or if the requested position was bad